util/src/main/scala/com/scalableminds/util/tools/JsonHelper.scala

```diff
@@ -96,8 +96,8 @@ object JsonHelper extends BoxImplicits with LazyLogging {
     override def reads(json: JsValue): JsResult[Option[T]] = json.validateOpt[T]
 
     override def writes(o: Option[T]): JsValue = o match {
-      case Some(t) ⇒ implicitly[Writes[T]].writes(t)
```

Comment:
`⇒` WAT?

Reply:
yeah, the unicode array was deprecated in 2.13 :sweat_smile:

Reply:
Good riddance.